apps/meeting/webpack.config.dev.js

```diff
@@ -1,3 +1,91 @@
-const config = require('./webpack.config.js');
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
 
-module.exports = Object.assign(config, { mode: 'development', devtool: 'inline-source-map' });
+const path = require('path');
+const InlineChunkHtmlPlugin = require('react-dev-utils/InlineChunkHtmlPlugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const app = 'meeting';
+
+module.exports = {
+  mode: 'development',
+  entry: ['./src/index.tsx'],
+  devtool: 'inline-source-map',
+  module: {
+    rules: [
```

Comment:
It would be great if we could reuse common rules, resolve, and other fields between dev and prod.